### PR TITLE
Add report view to build

### PR DESCRIPTION
**Pull request by eroshenkoam**

### CODE REVIEW

File: allure-e2e/pom.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>allure-core</artifactId>
+        <groupId>ru.yandex.qatools.allure</groupId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>allure-e2e</artifactId>
```
Comment (baev) [inline allure-e2e/pom.xml]: maybe `report-preview` or something else? It is not `e2e`

**Comment by baev** [comment]
retest this please

**Comment by baev** [comment]
retest this please

**Comment by eroshenkoam** [comment]
retest this please

**Comment by eroshenkoam** [comment]
retest this please

**Comment by eroshenkoam** [comment]
test this please

**Comment by eroshenkoam** [comment]
retest this please

**Comment by eroshenkoam** [comment]
retest this please

**Comment by baev** [comment]
Thanks!

**Pull request merged**